lib/ts/BufferWriterFormat.cc

```diff
@@ -755,6 +756,160 @@ bwf_register_global(std::string_view name, BWGlobalNameSignature formatter)
   return ts::bw_fmt::BWF_GLOBAL_TABLE.emplace(name, formatter).second;
 }
 
+BufferWriter &
+bwformat(BufferWriter &w, BWFSpec const &spec, bwf_errno const &e)
+{
+  constexpr std::array<std::string_view, 134> SHORT_NAME = {
```

Comment:
This feels fragile for two reasons:

1. 134 ?? Really, there's no way to autosize this??
2. These names might not be this consistent across all platforms we support.

Reply:
There's really no way to make the size automatic. If you get it wrong, it breaks during compile so it's not too bad. That's just the way `std::array` works.

I realize the names might not be completely consistent across platforms, but making it so would be some rather tricky Python code to grub through OS dependent include paths to find the raw data (presuming it's even in some sort of standard declaration style). I think this is the lesser of th two evils.